doc/md/cloud/getting-started.md

```diff
@@ -18,7 +18,7 @@ database schema.
 1. To get started with [Atlas Cloud](https://atlasgo.cloud/), create an account by clicking 'sign up' on the homepage.
 2. In the sign up screen, you can either sign up with GitHub or enter your work email and choose a password.
 3. Next, you will receive an email to verify your account. From your email, [Atlas Cloud](https://atlasgo.cloud/)
-will open in a new tab, and you will need to sign in to access your account.
+will open in a new tab, and you will need to sign in which will let you access your account.
```

Comment:
I'd love to use `sign-in` to match `checked-in` even though neither are technically right...